src/HOCON.Tests/Commas.cs

```diff
@@ -94,22 +114,31 @@ public void ThrowsParserExceptionOnMultipleCommasInArray()
          * FACT:
          * {a:1, b:2, c:3,} and {a:1, b:2, c:3} are the same object.
          */
-        [Fact(Skip = "Failed, not in spec")]
+        [Fact]
         public void ExtraCommaAtTheEndIgnored()
         {
             var hocon_1 = @"a:1, b:2, c:3,";
             var hocon_2 = @"a:1, b:2, c:3";
 
-            var config_1 = ConfigurationFactory.ParseString(hocon_1);
-            var config_2 = ConfigurationFactory.ParseString(hocon_2);
-            Assert.True(config_1.AsEnumerable().SequenceEqual(config_2.AsEnumerable()));
+            using (var config_1 = HoconParser.Parse(hocon_1).AsEnumerable().GetEnumerator())
+            {
+                using (var config_2 = HoconParser.Parse(hocon_2).AsEnumerable().GetEnumerator())
+                {
+                    while (config_1.MoveNext())
+                    {
+                        config_2.MoveNext();
```

Comment:
Because for this test, we're comparing 2 KeyValuePair from 2 different objects. The values of each KVP are HoconField, which does not have any IEquatable implementation. I'll add IEquatable implementation on the next commit.

Reply:
IEquatable added, you can use object1.AsEnumerable().SequenceEqual(object2.AsEnumerable()) now.